packages/cli/test/unit/utils.unit.js

```diff
@@ -469,6 +469,17 @@ describe('Utils', () => {
     });
   });
 
+  describe('validateNotExisting', () => {
+    it("returns true on the path contains the root folder '~'", () => {
+      expect(utils.validateNotExisting('~/try/pathFromRoot')).to.be.eql(
+        'The path should be based on current directory: ~/try/pathFromRoot.',
+      );
+    });
+    it("returns false on the path that folder's name contains hyphens", () => {
```

Comment:
@agnes512 

A hyphen is '-' . Where are the hyphens in this test?

Reply:
and the 'returns false...' name for the test...is incorrect because you are testing for 'True'.

Reply:
good catch!